src/types/package.rs

```diff
@@ -104,6 +105,7 @@ impl PackageType {
             PackageType::Nuget => ".NET",
             PackageType::Cargo => "Rust",
             PackageType::Golang => "Golang",
+            PackageType::Sbom => "Sbom",
```

Comment:
```suggestion
            PackageType::Sbom => "SBOM",
```

I think this is the "official" way to write it and that seems to be what we do with other languages here.